### DISCUSSION TITLE
How to create Webmin subdomain ?

### DISCUSSION THREAD

**Discussion by pol2095** [webmin/webmin#2091] [Q&A]
Hello,

I tried to create Webmin subdomain using this [tutorial](https://bobcares.com/blog/webmin-subdomain/).
I can connect to PHPMyAdmin using the url "https://mysite.com/phpmyadmin" but I want to connect it using "https://phpmyadmin.mysite.com/".

I added in OVH Zone DNS : 
`Domain: phpmyadmin.domain.com.    TTL: 0    type: CNAME    target: domain.com.`

in Webmin now 

1 - go to Servers > Apache Webserver.

2 - From that page, we select “create new virtual host”.

3 - Handle connections to address: We select the specific address... : "the IP address",
Server Name : "phpmyadmin.mysite.com",
Document Root: We enter the path of sub-domain : "/home/mysite/public_html/phpmyadmin"

1 - after Iadded the “A record” via Webmin. We go to Servers > BIND DNS Server.

2- Next, we selected the zone of the parent domain and clicked on “Add Record To Selected” button.

3 - In new Record Name field, we entered the subdomain name. "phpmyadmin"
In new record field, we entered the IP address.

when I try to connect to "phpmyadmin.mysite.com" on Google Chrome, I redirected to "www.phpmyadmin.mysite.com" and I have an error "DNS_PROBE_FINISHED_NXDOMAIN"

Thanks

**Comment by iliaross** [comment]
Hello,

This is not the best place to ask for help. Please consider using [forum.virtualmin.com](https://forum.virtualmin.com) instead.

**Reply by pol2095** [reply]
Sorry, I post it on the [forum](https://forum.virtualmin.com/t/how-to-create-webmin-subdomain/124947).
